src/appshell/view/preferences/noteinputpreferencesmodel.cpp

```diff
@@ -124,6 +124,7 @@ void NoteInputPreferencesModel::setPlayNotesWhenEditing(bool value)
 
     playbackConfiguration()->setPlayNotesWhenEditing(value);
     emit playNotesWhenEditingChanged(value);
+    playbackController()->toggleHearPlaybackWhenEditing();
```

Comment:
Having to call this method from here is not ideal. It would be better to make it so that that is not necessary.

Reply:
OK